hat/assets/js/apps/Iaso/domains/workflows/hooks/requests/useGetWorkflowVersions.ts

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable camelcase */
+import { UseQueryResult } from 'react-query';
+import { getRequest } from '../../../../libs/Api';
+import { useSnackQuery } from '../../../../libs/apiHooks';
+import { makeUrlWithParams } from '../../../../libs/utils';
+import {
+    WorkflowVersionsPaginated,
+    WorkflowsParams,
+    WorkflowVersionDetail,
+} from '../../types/workflows';
+
+const getWorkflowVersions = async (
+    options: WorkflowsParams,
+): Promise<WorkflowVersionsPaginated> => {
+    const { pageSize, entityTypeId, ...params } = options as Record<
```

Comment:
if accountId is added to params, it should be removed here in order not to be passed to the API